pkgs/desktops/elementary/wingpanel-indicators/notifications/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchFromGitHub, elementary, pkgconfig, meson, ninja, vala, gtk3, granite
+, wingpanel, libgee, dbus-glib, libwnck3, gobjectIntrospection, defaultIconTheme, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "wingpanel-indicator-notifications";
+  version = "2.1.1";
+
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "elementary";
+    repo = pname;
+    rev = version;
+    sha256 = "0kbpqwamh0y42ak4j5icdrjkbcy045dwc3wpwpya9a80ca9s0i2x";
+  };
+
+  passthru = {
+    updateScript = elementary.updateScript {
+      repoName = pname;
+    };
+  };
+
+  nativeBuildInputs = [
+    gobjectIntrospection
+    meson
+    ninja
+    pkgconfig
+    vala
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    dbus-glib
```

Comment:
Why is dbus-glib required? It is deprecated and gdbus from the glib library should be used instead.